.github/workflows/require-core-maintainer-approval.js

```diff
@@ -10,6 +10,10 @@ const CORE_MAINTAINERS = new Set([
 ]);
 
 module.exports = async ({ github, context, core }) => {
+  if (CORE_MAINTAINERS.has(context.payload.pull_request.user.login)) {
+    return;
+  }
```

Comment:
This change is for the following case:

1. A non-maintainer member reported a bug.
2. **_You_** filed a PR to fix the bug.
3. The non-maintainer member approved the PR.

Reply:
^ Are we doing this practice now? Imo code review should be done by someone who has knowledge about the code base, the reporter may have no knowledge.